src/main/java/dan/morefurnaces/items/ItemMoreFurnaces.java

```diff
@@ -0,0 +1,25 @@
+package dan.morefurnaces.items;
+
+import dan.morefurnaces.FurnaceType;
+import net.minecraft.block.Block;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+
+public class ItemMoreFurnaces extends ItemBlock {
+
+    public ItemMoreFurnaces(Block block) {
+        super(block);
+        setMaxDamage(0);
+        setHasSubtypes(true);
+    }
+
+    @Override
+    public int getMetadata(int i) {
+        return i;
```

Comment:
Is returning `i` correct here, or should it be returning the position in the Furnace Types Enum?

Reply:
This override is just needed to allow metadata to be used, as in the `Item` class in MC, it has:
```
public int getMetadata(int damage) {
    return 0;
}
```
So this is just a passthrough override